package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "taskcluster-lib-pulse",
-  "version": "1.1.0",
+  "version": "2.0.0",
```

Comment:
Please revert this bit.  I will do a 2.0.0 release on master after merging, at which time I'll update package.json, make a git tag, and upload the release.